gradle/posix.gradle

```diff
@@ -72,7 +72,7 @@ kotlin {
             }
             disableCompilation(crossCompileTargets)
             configure([mingwX64Test]) {
-                environment "PATH": "C:\\msys64\\mingw64\\bin;C:\\Tools\\msys64\\mingw64\\bin"
+                environment "PATH": "C:\\msys64\\mingw64\\bin;C:\\Tools\\msys64\\mingw64\\bin;C:\\Tools\\msys2\\mingw64\\bin"
```

Comment:
Is there a way to find its location without hardcoding directories?

Reply:
Nope. It's from our experience :(

Reply:
Ok. 
Then I have 2 concerns here:
1. Shouldn't we add a property or env variable that users can specify in case they have non-default path in their system. Something like: `System.findProperty(MINGW64_PATH) ?: DEFAULT_MINGW64_PATH`
2. Nit: There are at least 2 places with that string. Maybe move to some const?